edkahara/iReporter
# Ft create v2 login endpoint #162422843

__What does this PR do__
Create a login endpoint for iReporter v2.

__Description of tasks to be completed.__
Create an api endpoint that facilitates the logging in of a user. Test the api endpoint.

__How should it be manually tested?__
Clone the project. Check out the ft-create-v2-login-endpoint-#162422843 branch. Start the environment using `env\Scripts\activate` and `env.bat` to set environment variables. Run flask run on the terminal and go to Postman. To perform the tests run nosetests.

__Relevant pivotal tracker stories.__
#162422843
